src/components/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import {useEffect, useState} from 'react'
+import axios from 'axios'
+
+function Comments(article_id){
+
+const [comments, setComments] = useState([])
+
+const articleId = article_id.article_id
+    useEffect(() => {
+    axios.get(`https://northcoders-news-app-ei5k.onrender.com/api/articles/${articleId}/comments`)
+    .then(({data}) => {
+        setComments(data.comments)
+        console.dir(data.comments)
+    })
+}, [])
+
+    return (
+        <>
+        <h2>Comments:</h2>
+        <ul>
+            {comments.map((comment) => {
+                return(
+                    <li key={comment.comment_id} className="articleHome">
+                        <p>{comment.body}</p>
+                        <p>{comment.author} | Posted: {comment.created_at}</p>
```

Comment:
It would be better to have the timestamp formatted in a bit more of a human-readable way, than the default. Have a think about how you could manipulate the data yourself to do this or look into some packages that can do it for you 💡